internal/app/wwclient/root.go

```diff
@@ -31,14 +31,16 @@ var (
 		RunE:         CobraRunE,
 		SilenceUsage: true,
 	}
-	DebugFlag bool
-	PIDFile   string
-	Webclient *http.Client
+	DebugFlag       bool
+	PIDFile         string
+	Webclient       *http.Client
+	WarewulfConfArg string
```

Comment:
It looks to me like this PR is actually adding configuration to specify the config file, not adjusting for #781. Have I missed something?

Reply:
I assume that since #781 a configuration file has to be specified, before it was specified implicitly. But this could also have been introduced by me earlier. Another fix would be to add to warewulf conf.Get() a implicit default configuration file, but as Get is only called twice, besides the tests, I think this is the better solution.

Reply:
Agreed. Thanks for the clarification.